<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="114093" started-at="2018-02-15T15:27:12Z" finished-at="2018-02-15T15:29:06Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="114093" started-at="2018-02-15T15:27:12Z" finished-at="2018-02-15T15:29:06Z">
      <class name="assignment.ebay.testcases.LogInAndPurchase">
        <test-method status="PASS" signature="init()[pri:0, instance:assignment.ebay.testcases.LogInAndPurchase@5e0e82ae]" name="init" is-config="true" duration-ms="12146" started-at="2018-02-15T15:27:13Z" finished-at="2018-02-15T15:27:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="LogInAndPurchaseAProductAddInCart(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:assignment.ebay.testcases.LogInAndPurchase@5e0e82ae]" name="LogInAndPurchaseAProductAddInCart" duration-ms="76670" started-at="2018-02-15T15:27:25Z" data-provider="LogInAndPurchaseData" finished-at="2018-02-15T15:28:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[assignment@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ebay@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[books]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[1234567812345678]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[123]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[09/20]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at assignment.ebay.testcases.LogInAndPurchase.LogInAndPurchaseAProductAddInCart(LogInAndPurchase.java:85)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogInAndPurchaseAProductAddInCart -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:assignment.ebay.testcases.LogInAndPurchase@5e0e82ae]" name="tearDown" is-config="true" duration-ms="907" started-at="2018-02-15T15:28:42Z" finished-at="2018-02-15T15:28:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="init()[pri:0, instance:assignment.ebay.testcases.LogInAndPurchase@5e0e82ae]" name="init" is-config="true" duration-ms="10680" started-at="2018-02-15T15:28:43Z" finished-at="2018-02-15T15:28:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="LogInAndPurchaseAProductBuyItNow(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:assignment.ebay.testcases.LogInAndPurchase@5e0e82ae]" name="LogInAndPurchaseAProductBuyItNow" duration-ms="12057" started-at="2018-02-15T15:28:53Z" data-provider="LogInAndPurchaseData" finished-at="2018-02-15T15:29:05Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[assignment@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ebay@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[books]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[1234567812345678]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[123]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[09/20]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at assignment.ebay.testcases.LogInAndPurchase.LogInAndPurchaseAProductBuyItNow(LogInAndPurchase.java:65)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogInAndPurchaseAProductBuyItNow -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:assignment.ebay.testcases.LogInAndPurchase@5e0e82ae]" name="tearDown" is-config="true" duration-ms="907" started-at="2018-02-15T15:29:05Z" finished-at="2018-02-15T15:29:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- assignment.ebay.testcases.LogInAndPurchase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
